docs/source/glance.rst

```diff
@@ -122,25 +122,25 @@ Here's the whole picture of the code:
            extensions.PlotReport(
                ['main/accuracy', 'validation/main/accuracy'],
                'epoch', file_name='accuracy.png'))
-   
+
    # Print selected entries of the log to stdout
    trainer.extend(extensions.PrintReport(
        ['epoch', 'main/loss', 'validation/main/loss',
         'main/accuracy', 'validation/main/accuracy', 'elapsed_time']))
-   
+
    #  Run the training
    trainer.run()
-   
+
    x, t = test[np.random.randint(len(test))]
 
    predict = model.predictor(x[None]).data
    predict = predict[0][0]
-   
+
    if predict >= 0:
       print('Predicted Poisonous, Actual ' + ['Edible', 'Poisonous'][t[0]])
    else:
       print('Predicted Edible, Actual ' + ['Edible', 'Poisonous'][t[0]])
-   
+
 .. testoutput::
```

Comment:
Strictly speaking it's not related to this PR, but could you add `:hide:` here? It looks odd.